apps/frontend/src/components/RequestModal/RequestModal.tsx

```diff
@@ -84,13 +88,13 @@ export default function RequestModal({ show, handleCloseModal, request }: Props)
         <Form.Control type="hidden" name="request_id" value={Number(request.id)} />
       </Form.Group>
       <Container className={styles.btnContainer} fluid>
-        <Row className="gx-3 gy-2">
-          <Col sm={6}>
+        <Row className={`gx-3 gy-2 ${styles.responseBtnRow}`}>
```

Comment:
Where did you find these classNames? I couldn't find them within React Boostrap docs

Reply:
Here: https://getbootstrap.com/docs/4.0/utilities/spacing/